clang/test/CheckedCRewriter/alloc_type_param.c

```diff
@@ -1,8 +1,6 @@
 // RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
 // RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
-// RUN: cconv-standalone -output-postfix=checkedNOALL %s
-// RUN: %clang -c %S/alloc_type_param.checkedNOALL.c
-// RUN: rm %S/alloc_type_param.checkedNOALL.c
+// RUN: cconv-standalone %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
```

Comment:
I'm very confused. Did this test not pass before so you had to delete these lines? That would mean someone checked in a version of the compiler that didn't pass the tests. If that's so, we need to go back to when that happened and figure out what changed.

Reply:
The three lines above generate the converted file and then compiles it, whereas the line that I added does all of that in one line by reading directly from stdin and compiling. I came across a way to do it one line in a test someone had checked in (thanks!) and realized it was more descriptive and brilliant than the clunky way I was doing it, so that's what this is. Once the addcr PR comes in, I'm going to modify the testgenerator script to do the same, so all of our tests will look like this for the compilation phase.

Reply:
Got it. Very nice! Can you make this change in other places too? I didn't see the change being done universally in this PR.

Reply:
I made this change in all files that aren't autogen'd via a recursive grep. I believe the only files with this outdated form of compiling are the autogened tests, and I thought it'd be a very easy thing to tack on with the addcr changes, since that's the one big change that'll happen to the auto-gen'd tests. If it makes more sense here, I can make those changes in my next commit for this PR. Do we want that?